pymobiledevice3/tunneld.py

```diff
@@ -296,6 +296,24 @@ async def close(self) -> None:
             with suppress(asyncio.CancelledError):
                 await task
 
+    def get_tunnels_ips(self) -> Dict:
+        """ Retrieve the available tunnel tasks and format them as {UUID: IP} """
```

Comment:
```suggestion
        """ Retrieve the available tunnel tasks and format them as {UDID: [IP]}  """
```

Reply:
Done in [9690655](https://github.com/doronz88/pymobiledevice3/pull/1112/commits/96906550d20d1cbbd6baf511cdec9e9fbc0f8a49)